AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -3630,7 +3630,7 @@ bosch-mobility-solutions.com#%#document.cookie = "Privacy_LVL=_pcbc%3D3";
 bosch-presse.de#%#document.cookie = "bosch-privacy-settings-v2={%22comfort%22:0%2C%22marketing%22:1%2C%22processing%22:%223020-02-03T15:03:09.068Z%22}";
 galaxus.de#%#document.cookie = "cookieMessageAccepted=1";
 radeberger-gruppe.de#%#document.cookie = "dataPrivacy-acceptCookies=confirmed";
-voelkner.de,digitalo.de#%#AG_onLoad(function() { setTimeout(function() { var el = document.querySelector('#cookie_banner_content > .btn.btn-cookie-consent'); if(el) el.click(); }, 300); });
+voelkner.de,digitalo.de#%#AG_onLoad(function() { setTimeout(function() { var el = document.querySelector('#cc_info > .btn.btn-cookie-consent'); if(el) el.click(); }, 300); });
```

Comment:
Rules for FF/Opera missing (they cannot get javacript rule updates, user has to wait for next Extension version).